ui/client/components/graph/node-modal/editors/expression/EditorType.tsx

```diff
@@ -0,0 +1,58 @@
+import BoolEditor from "./BoolEditor"
+import RawEditor from "./RawEditor"
+import StringEditor from "./StringEditor"
+import FixedValuesEditor from "./FixedValuesEditor"
+import _ from "lodash";
+
+export enum dualEditorMode {
+    SIMPLE = "SIMPLE",
+    RAW = "RAW",
+}
+
+export enum editorTypes {
+  RAW_PARAMETER_EDITOR =  "RawParameterEditor",
+  BOOL_PARAMETER_EDITOR =  "BoolParameterEditor",
+  STRING_PARAMETER_EDITOR =  "StringParameterEditor",
+  FIXED_VALUES_PARAMETER_EDITOR =  "FixedValuesParameterEditor",
+  DUAL_PARAMETER_EDITOR = "DualParameterEditor",
+}
+
+export const editors = {
```

Comment:
defined type for editors would prevent adding wrong props

Reply:
Done